src/DeckViewer.jsx

```diff
@@ -1,76 +1,106 @@
-import { useState } from 'react';
+import { useMemo, useState } from 'react';
```

Comment:
It doesn't seem like this component is even accessible anymore, why so many changes?